app/controllers/concerns/exception_handler.rb

```diff
@@ -0,0 +1,27 @@
+module ExceptionHandler
```

Comment:
This is the concern for prettier message, usually i have a long list of different types of exceptions here, but for the sake of this small project 2 main exceptions + generic exception handler were added